src/views/SessionView/Whiteboard.vue

```diff
@@ -278,9 +278,26 @@ export default {
       // convert file to base64
       reader.readAsDataURL(file);
       reader.onload = () => {
-        this.zwibblerCtx.createNode("ImageNode", {
+        const img = new Image();
+        const node = this.zwibblerCtx.createNode("ImageNode", {
           url: reader.result
         });
+        img.src = reader.result;
+        img.onload = () => {
+          const whiteboard = document.querySelector(".whiteboard-container");
+          const whiteboardWidth = whiteboard.clientWidth;
+          const whiteboardHeight = whiteboard.clientHeight;
+          let scaleFactor = 1;
+
+          // scale image below the whiteboard width and height
+          if (img.width > whiteboardWidth) {
+            scaleFactor = 1 / (img.width / whiteboardWidth + 1);
+            this.zwibblerCtx.scaleNode(node, scaleFactor, scaleFactor);
```

Comment:
@jkeat,

I couldn't find anything with autoscaling on Zwibbler, so as one implementation I decided to just check if the image is greater than the width or height of the whiteboard and scale it down by how much greater the image was